### PR TITLE
TextField MultiValueCharFilter filtering implemented in BaseFilterSet…

### PR DESCRIPTION
**Pull request by briddo**
… along with tests

### Fixes: #477 

I added MultiValueCharFilter to FILTER_DEFAULTS for a TextField and wrote appropriate tests.

### CODE REVIEW

**Comment by glennmatthews** [comment]
Can you confirm (manually is fine) that the REST API filtering on a TextField still accepts filters that worked before this change?

**Comment by briddo** [comment]
> Can you confirm (manually is fine) that the REST API filtering on a TextField still accepts filters that worked before this change?

I just checked and couldn't find any filters which reference a TextField. But I checked the CharField filters and it works the same with just one value and/or many values, and this uses the same MultiValueCharFilter as the TextField has been setup to use.

File: nautobot/utilities/tests/test_filters.py [hunk 1]
```diff
@@ -125,6 +126,7 @@ class DummyFilterSet(BaseFilterSet):
         multiplechoicefield = django_filters.MultipleChoiceFilter(field_name="choicefield")
         multivaluecharfield = MultiValueCharFilter(field_name="charfield")
         tagfield = TagFilter()
+        textfield = MultiValueCharFilter(field_name="textfield")
```
Comment (glennmatthews) [inline nautobot/utilities/tests/test_filters.py:129]: This shouldn't be needed here, right? Isn't the whole idea of this PR that the correct filter class will get automatically chosen for the TextField?

**Review by jathanism — APPROVED** [review]
🚢 🇮🇹

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**